src/Api/UI/Widget/FlashMessage.php

```diff
@@ -26,12 +31,13 @@ public function run(): string
 
         foreach ($flashes as $type => $data) {
             foreach ($data as $message) {
-                $html .= Message::widget()
-                    ->headerColor($type)
-                    ->headerMessage($message['header'] ?? '')
+                $html .= Alert::widget()
+                    // ->heading($message['header'] ?? '')
```

Comment:
Delete it if it's not used?

Reply:
I hope we can use this in the future when the Alert widget supports it

Reply:
https://getbootstrap.com/docs/5.0/components/alerts/#additional-content